Stdlib/Cairo/Ec.juvix

```diff
@@ -0,0 +1,127 @@
+--- Functions for various actions on the STARK curve:
+--- y^2 = x^3 + alpha * x + beta
+--- where alpha = 1 and beta = 0x6f21413efbe40de150e596d72f7a8c5609ad26c15c915c1f4cdfcb99cee9e89.
+--- The point at infinity is represented as (0, 0).
+--- Implementation largely follows:
+--- https://github.com/starkware-libs/cairo-lang/blob/master/src/starkware/cairo/common/ec.cairo
+module Stdlib.Cairo.Ec;
+
+import Stdlib.Data.Field open;
+import Stdlib.Data.Bool open;
+
+module StarkCurve;
+  ALPHA : Field := 1;
+
+  BETA : Field :=
+    3141592653589793238462643383279502884197169399375105820974944592307816406665;
+
+  ORDER : Field :=
+    3618502788666131213697322783095070105526743751716087489154079457884512865583;
```

Comment:
Can we use the new hex literals for these?

https://github.com/anoma/juvix/pull/2746